File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		long num;
		int i = 0, j = 0;
		System.out.print("Enter num : ");
		Scanner sc = new Scanner(System.in);
		num = sc.nextLong();
		while(num != 0) {
			if((num % 10) % 2 == 1)
				i++;
			else
				j++;
			num = num / 10;
		}
		System.out.println("odd : " + i + "\neven : " + j);
	}
}

File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/Python/prog5.py
num = int(input())
i = 0
j = 0
while(num != 0):
    if((num % 10) % 2 == 1):
        i += 1
    else:
        j += 1
    num = int(num / 10)

print("odd : ", i, "\neven : ", j)
